libvast_test/src/node.cpp

```diff
@@ -28,7 +28,8 @@ namespace fixtures {
 
 node::node() {
   MESSAGE("spawning node");
-  test_node = self->spawn(system::node, "test", directory / "node");
+  test_node = self->spawn(system::node, "test", directory / "node",
+                          std::chrono::milliseconds::zero());
```

Comment:
Let's add a comment that we're using `zero()` to disable hard kills in unit tests here. (maybe even with a link to the discussion in this review)